pkg/controller.v2/controller_logger.go

```diff
@@ -15,32 +15,31 @@
 package controller
```

Comment:
Do you want to use the logger? Then I think we could put the code in a separate package.